web_frontend/config/urls.py

```diff
@@ -10,4 +10,5 @@
     url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework')),
     url(r'^api/', include('osmaxx.excerptexport.rest_api.urls', namespace='excerptexport_api')),
     url(r'^job_progress/', include('osmaxx.job_progress.urls', namespace='job_progress')),
+    url(r'^pages/', include('osmaxx.core.urls', namespace='pages')),
```

Comment:
We combine them, but if you look at our current download page, we already have a lot of content there when logged in, so I don't think that would that helpful. The `pages` naming isn't great, I agree, but I don't want to have `static_pages` since they are not static for the user, and `cms` is also wrong ;-) .

They are static pages for the user, as the rest, and since we already register the excerpt exports urls[1] globally, that option is out of question as well.

Cleaner approach would be:

- include `excerpt_export` urls in `core.urls`
- inlcude `core.urls` as top level urls
- combine both download pages and push the static downloads to the right side (ie. 4 of 12 columns?) when logged in

[1]: top level registered urls
```
url(r'', include('osmaxx.excerptexport.urls', namespace='excerptexport')),
...
url(r'', include('social.apps.django_app.urls', namespace='social')),
```